astropy/cosmology/scalar_inv_efuncs.pyx

```diff
@@ -0,0 +1,225 @@
+""" Cython inverse efuncs for cosmology integrals"""
+#cython boundcheck=False
```

Comment:
No, it actually helps, even with lists.  Okay, not very much, but I thought it might make ndarrays faster than lists, and it doesn't...